lib/core/src/lib/language-menu/language-menu.component.spec.ts

```diff
@@ -113,4 +113,20 @@ describe('LanguageMenuComponent', () => {
         component.changeLanguage(languages[2]);
         expect(changedLanguageSpy).toHaveBeenCalledWith(languages[2]);
     });
+
+    it('should update HTML lang attribute when language is changed', () => {
```

Comment:
This test should be written for file where you added solution so it should be written in test file for language.service.ts. If there is no spec file for this file then please create one and put test there